scarb/tests/build.rs

```diff
@@ -41,11 +43,18 @@ fn compile_simple() {
         vec![".fingerprint", "hello.sierra.json", "incremental",]
     );
 
+    let hashed_manifest_path = hash_path(
+        fsx::canonicalize(t.child("Scarb.toml"))
+            .unwrap()
+            .to_str()
+            .unwrap(),
+    );
+
     cache_dir
-        .child("registry/std")
+        .child(format!("{hashed_manifest_path}/registry/std"))
```

Comment:
Isn't this kind of besides the point? 
I **do want** to share the downloaded registry packages between workspaces.